### PR TITLE
Add of mobile navbar

### LINKED ISSUE #12: Add hamberguer menu
We should hide the menu and show a hamburger menu on mobile
image

### PR DESCRIPTION
**Pull request by LeonelSantiago**
- Add of hamburger menu style for mobile navbar
- Change of styles in the current desktop navbar
- Call of materialize AutoInit() function when component is mounted
- Add of class hide-on-small-only on Nav text
- Add of class sidenav-close on mobile-nav to hide when an element is clicked
- Fix of vertical aligment
- Adjust on font-size on smaller devices

Closes #12

### CODE REVIEW

File: src/components/Nav.vue [hunk 1]
```diff
@@ -1,68 +1,68 @@
...
+    <nav>
+      <div class="nav-wrapper blue darken-1">
+        <div class="logo">
+          <div class="brand-logo center">
+            <svg viewBox="0 0 48 48">
+              <path
+                d="M39.13,39V26.18l0,0H32l0,0V43.74l0,0A21.15,21.15,0,0,1,27.57,45a0,0,0,0,1,0,0V32.2l0,0H20.47l0,0V45.29l0,0H16l0,0V35.75l0,0H8.88l0,0v9.54l0,0H2.71l0,0V2.71l0,0h21A21.47,21.47,0,0,1,45.29,24.9,21.24,21.24,0,0,1,39.16,39,0,0,0,0,1,39.13,39Z"
+              />
+            </svg>
+            <span>Dominican Open Source</span>
```
Comment (DannyFeliz) [inline src/components/Nav.vue:12]: We want to hide this when this when the windows width is < ~600px
```suggestion
            <span class="hide-on-small-only">Dominican Open Source</span>
```

File: src/components/Nav.vue [hunk 2]
```diff
@@ -1,68 +1,68 @@
...
+  color: #fff;
+  font-size: 1.9rem;
+  svg {
+    fill: currentcolor;
+    width: 1.5em;
+    margin-right: 8px;
+    margin-top: 10px;
+  }
+  span {
+    font-size: 2.1rem;
```
Comment (DannyFeliz) [inline src/components/Nav.vue:65]: Add a media query to change the font size to `1.6rem` when the window width is <= 768px (tablet size)

**Comment by DannyFeliz** [comment]
Some suggestions:
- I would like to close the menu on click on a menu item.
- The logo and the title should be aligned to the center.

### Before the change
image

### After the change
image

File: src/components/Nav.vue (hunk 1 above)
Comment (LeonelSantiago) [reply src/components/Nav.vue]: Done in the last commit.

File: src/components/Nav.vue (hunk 2 above)
Comment (LeonelSantiago) [reply src/components/Nav.vue:66]: Done in the last commit.

File: src/components/Nav.vue [hunk 3]
```diff
@@ -3,14 +3,14 @@
     <nav>
       <div class="nav-wrapper blue darken-1">
         <div class="logo">
-          <div class="brand-logo center">
+          <router-link to="/" class="brand-logo center">
```
Comment (DannyFeliz) [inline src/components/Nav.vue:6]: Redirect directly to `PopularRepositories`
```suggestion
          <router-link :to="{ name: 'PopularRepositories' }" class="brand-logo center">
```

**Comment by LeonelSantiago** [comment]
@DannyFeliz Just to confirm. I think you tried to say that the logo and title should be aligned to the left. Is that correct?

**Comment by LeonelSantiago** [comment]
Please review the last changes.

**Comment by DannyFeliz** [comment]
LGTM.
Thanks.

**Pull request merged**